examples/ligand_in_water/ligand_in_water.ipynb

```diff
@@ -36,8 +36,7 @@
     "from openff.units import unit\n",
```

Comment:
<p>Obviously I'm biased, but I'd suggest doing this at the same time as the box vectors (which are kinda from one perspective a bit useless without positions anyway) and using the new <code>Topology.set_positions()</code> method. This also reduces the duplicated work in running TIP4P.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openforcefield/openff-interchange/pull/513/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>